setup.py

```diff
@@ -0,0 +1,22 @@
+import os
+from setuptools import setup, find_packages
+
+BASE = os.path.dirname(os.path.abspath(__file__))
+
+setup(
+    name="intake-nested-yaml-catalog",
+    setup_requires=["vcver", "setuptools-parcels"],
```

Comment:
Reading up on versioneer, it produces it's own concept of version string. I'd suggest using one or the other exclusively, to avoid a mismatch inversions.